src/classes/STG_InstallScript_TEST.cls

```diff
@@ -32,7 +32,8 @@
 * @date 2014
 * @group Settings
 * @description Test Class for the Install Script for the NPSP package. 
-*/@isTest
+*/
+@isTest
```

Comment:
Do we have to change our test methods to use the testMethod annotation instead of @isTest for apexDocs purposes?